components/model.py

```diff
@@ -926,6 +925,104 @@ def get_prerequisite(module_code):
     return DB_CURSOR.fetchall()
 
 
+def get_prerequisite_as_string(module_code):
+    '''
+        Returns a string of pre-requisites of specified module_code
+    '''
+    prerequisites = get_prerequisite(module_code)
+    prereq_list = convert_to_list(prerequisites)
+
+    # sort list of lists based on index (which is the first elem of each row)
+    prereq_list.sort(key=lambda row: row[INDEX_FIRST_ELEM])
+
+    prereq_string = convert_list_of_prereqs_to_string(prereq_list)
+
+    return prereq_string
+
+
+def convert_list_of_prereqs_to_string(prereq_list):
```

Comment:
There is something wrong with my prerequisitePopulator.sql, will push a fix soon